### ISSUE TITLE
"comment" is a keyword in PostgreSQL - export fails due to SQL

### ISSUE THREAD

**Issue by mudrd8mz** [academic-moodle-cooperation/moodle-mod_checkmark#9]
The query at line 3178 of locallib.php contains something like

```
SELECT ... MAX(s.submissioncomment) comment ...
```

This fails on PostgreSQL because `comment` is a non-reserved keyword in this DB. "Non-reserved key words only have a special meaning in particular contexts and can be used as identifiers in other contexts." (http://www.postgresql.org/docs/9.2/static/sql-keywords-appendix.html)

To fix this, the alias could be renamed. But much better (and easier) way is to follow https://docs.moodle.org/dev/SQL_coding_style and always use `AS` keyword for column aliases:

```
SELECT ... MAX(s.submissioncomment) AS comment ...
```

which then works correctly.

**Comment by kaspot** [comment]
added to our todo list

**Comment by kaspot** [comment]
Should be fixed in the newest Moodle 2.8 version.
